can/dbc.cc

```diff
@@ -1,27 +1,251 @@
+#include <algorithm>
+#include <cmath>
+#include <filesystem>
+#include <fstream>
+#include <map>
+#include <pwd.h>
+#include <regex>
+#include <set>
+#include <sstream>
+#include <unistd.h>
 #include <vector>
 
 #include "common_dbc.h"
 
-std::vector<const DBC*>& get_dbcs() {
-  static std::vector<const DBC*> vec;
-  return vec;
+std::string dbc_file_path;
```

Comment:
why is this a global?

Reply:
doesn't need to be, thought `set_dbc_file_path` might've been slow but it takes 0.001396 ms